Android/app/src/main/java/com/example/samplestickerapp/AddStickerPackActivity.java

```diff
@@ -123,7 +123,7 @@ private void launchWhatsAppPlayStorePage() {
                 final boolean smbAppInstalled = WhitelistCheck.isPackageInstalled(WhitelistCheck.SMB_WHATSAPP_PACKAGE_NAME, packageManager);
                 final String playPackageLinkPrefix = "http://play.google.com/store/apps/details?id=";
                 if (whatsAppInstalled && smbAppInstalled) {
-                    launchPlayStoreWithUri("https://play.google.com/store/apps/developer?id=WhatsApp+Inc.");
+                    launchPlayStoreWithUri("https://play.google.com/store/apps/developer?id=WhatsApp+LLC");
```

Comment:
This is not related to the PR description, but definitely has to be fixed. Good catch!

Reply:
Ya... I just find that after made PR and don't want to make it as separate Pull Request. So, just commit with that request.